pkg/imrpc/proxy.proto

```diff
@@ -0,0 +1,344 @@
+syntax="proto3";
+
+package imrpc;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "github.com/longhorn/longhorn-engine/proto/ptypes/controller.proto";
+import "github.com/longhorn/longhorn-engine/proto/ptypes/syncagent.proto";
+
+service ProxyService {
```

Comment:
Can we just rename to `ProxyEngineService`?

Reply:
Updated.